### PR TITLE
Feature/tribe v3

### LINKED ISSUE #7552: [DESIGN+BUILD] Turn off Tribe tab in current  orgs
### User Story

As an org owner, an not having sponsored a hackathon, I should not have the tribe tab on my orgs page.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: The "Tribe" tab in normal orgs page (example: https://gitcoin.co/cryptorado-community) contains similar info with the "People" and "Followers" tab, so it would be easier for users to browse if we consolidate them.

### Description

• Put Following and Followers (under current Followers tab) into the People tab
• Remove Tribes and Followers tab
• Which means under People tab, we have four sub-tabs: Core Team, Followers, Following, and Leaderboard. Keep the Leaderboard view. Core Team can view can be the same as Followers/Following (with Chat, Invite to Bounty buttons)
orgs page

### Current Behavior
[comment]: # (Describe what actually happened.)
Tribe, Followers, People tab show up in non-tribes org page

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Consolidate to People tab

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- [x] Put Following and Followers (under current Followers tab) into the People tab
- [x] Remove Tribes and Followers tab
- [x] Which means under People tab, we have four sub-tabs: Core Team, Followers, Following, and Leaderboard. Keep the Leaderboard view. Core Team can view can be the same as Followers/Following (with Chat, Invite to Bounty buttons)

### PR DESCRIPTION
**Pull request by sanchaymittal**
##### Description

Video Demo: https://www.loom.com/share/1955e703edf5495e818f16284169b541

- Put Following and Followers (under current Followers tab) into the People tab
- Remove Tribes and Followers tab
- Four sub-tabs: Core Team, Followers, Following & Leaderboard
- Keep the Leaderboard view.
- Core Team can view can be the same as Followers/Following (with Chat)

##### Refers/Fixes

#7552

### CODE REVIEW

**Review by PixelantDesign — COMMENTED** [review]
Lookin' good! Could we order the tabs

Leaderboard, Core Team, Followers & Following

**Comment by thelostone-mc** [comment]
@PixelantDesign I see the suggest bounty feature is removed as well ?
If this is intended -> @sanchaymittal we would have to remove that code as well

either that or it has to be pushed to it's own tab

**Comment by sanchaymittal** [comment]
> @PixelantDesign I see the suggest bounty feature is removed as well ?
> If this is intended -> @sanchaymittal we would have to remove that code as well
> 
> either that or it has to be pushed to it's own tab

Code is removed, it was present in tab_tribe: https://github.com/gitcoinco/web/pull/7595/files#diff-173d59f8e2d92e30a5f56a9ec0830c7fL60

**Comment by thelostone-mc** [comment]
@sanchaymittal I'm looking at the PR and I still see suggest_bounty.html in the codebase which would now be dead code
Additionally, all the CSS + JS associated with it still present 
All of these would need to be weeded out if the feature is being retired

**Comment by sanchaymittal** [comment]
> @sanchaymittal I'm looking at the PR and I still see `suggest_bounty.html` in the codebase on your branch which would now be dead code
> Additionally, all the CSS + JS associated with it still present
> All of these would need to be weeded out if the feature is being retired

Looking into this...

**Comment by sanchaymittal** [comment]
> > @sanchaymittal I'm looking at the PR and I still see `suggest_bounty.html` in the codebase on your branch which would now be dead code
> > Additionally, all the CSS + JS associated with it still present
> > All of these would need to be weeded out if the feature is being retired
> 
> Looking into this...

**Comment by sanchaymittal** [comment]
@thelostone-mc Removed the dead code.

File: app/dashboard/templates/profiles/tab_people.html [hunk 1]
```diff
@@ -1,27 +1,193 @@
...
-        <div class="activity-avatar ml-5">
-          <img height="24" width="24" class="avatar secondary_avatar rounded-circle align-self-end" src="{% avatar_url org %}" />
-        </div>
-        <div class="tab-projects__item-details ml-5 my-auto w-100">
-          <div class="font-body d-inline">
-            <a class="font-body" href="/profile/{{org}}">
-              {{org}}
+<div class="tab-projects d-flex flex-column">
+
+  {% if foltab == 'core_team' %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_people.html:23]: Is this needed?

File: app/dashboard/templates/profiles/tab_people.html [hunk 2]
```diff
@@ -1,27 +1,193 @@
...
+    <!-- This shows core team of Org-->
+  {% if team|length == 0 %}
+    <p class="text-center">No Members</p>
+  {% endif %}
+  {% for member in team %}
+  <div class="tab-projects__item d-flex mb-0 pt-1 pb-1 bounty_row">
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_people.html:29]: Indent by 1

File: app/dashboard/templates/profiles/tab_people.html [hunk 3]
```diff
@@ -1,27 +1,193 @@
...
+          {% for follower in followers %}
+            {% if forloop.last %}
+              <img class="rounded-circle" style="height: 25px; width: 25px;" src="/dynamic/avatar/{{ follower.org.handle }}" alt="">
+            {% else %}
+              <img class="rounded-circle" style="height: 25px; width: 25px; margin-right: -12px " src="/dynamic/avatar/{{ follower.org.handle }}" alt="">
+            {% endif %}
+          {% endfor %}
+        </div>
+        {% if followers|length >= 1 %}
+        <p class="text-muted font-smaller-4 my-0">Followed by {% for follower in followers %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/tab_people.html:62]: Indent

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Indent fixes needed! I just left comments on a few but there are a few more
~Cant seem to find the css rules being removed either!~

Are we sure that all the css rules are being used?

**Review by PixelantDesign — APPROVED** [review]

File: app/dashboard/templates/profiles/tab_people.html (hunk 1 above)
Comment (sanchaymittal) [reply app/dashboard/templates/profiles/tab_people.html:23]: Yes, I think it should be renamed to something common as all the other tabs are using the same css styling too 
Screenshot 2020-10-12 at 7 41 07 PM

File: app/dashboard/templates/profiles/tab_people.html (hunk 2 above)
Comment (sanchaymittal) [reply app/dashboard/templates/profiles/tab_people.html:29]: Done

**Review by thelostone-mc — APPROVED** [review]

**Review by octavioamu — APPROVED** [review]

**Pull request merged**